content/introduction/usage.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Basic Usage
-weight: 3
+title: Getting Started
+weight: 2
 ---
 
 ## Install IPFS
 
-If you haven't done so, your first step is to **install IPFS**! Most people prefer to install a prebuilt package - which you can do on the [IPFS distributions page](https://dist.ipfs.io/#go-ipfs) by clicking "Install go-ipfs" (our reference implementation written in Go) and then following the instructions for [installing from a prebuilt package](../install/#installing-from-a-prebuilt-package).
+If you haven't done so, your first step is to **install IPFS**! Most people prefer to install a prebuilt package — which you can do on the [IPFS distributions page](https://dist.ipfs.io/#go-ipfs) by clicking "Install go-ipfs" (our reference implementation written in Go) and then following the instructions for [installing from a prebuilt package](../install/#installing-from-a-prebuilt-package).
```

Comment:
"If you haven't already done so" (clarity)
One thing ... I'm a bit concerned that people will land on this page who aren't developers but who might benefit from just downloading ipfs-desktop and playing with it. Can we add some instructions for those folks here too?

Reply:
For sure! But in a different PR would be best -- I didn't really touch the content of the `usage.md` file (just needed it understand what was on it so I could usefully think about improving the nav structure). I'm sure it could use a going-over that will result in many clarity changes.

Reply:
Maybe just add a sentence on the end of this like so? Kind of an escape clause :)
Or, if you just want to give the desktop-app implementation of IPFS a go, and not worry about the command line at all, you can [get started with that here](https://github.com/ipfs-shipyard/ipfs-desktop).

Reply:
OK ok ok  :) Good point. _It is done_ -->
<img width="844" alt="Screenshot 2019-05-21 10 43 53" src="https://user-images.githubusercontent.com/4827522/58118288-5f51bc80-7bb5-11e9-9725-2c15207bdde3.png">